types/forms/index.d.ts

```diff
@@ -1,7 +1,13 @@
 // Type definitions for forms 1.3
 // Project: https://github.com/caolan/forms
 // Definitions by: suXin <https://github.com/suXinjke>
+//                 Pelle Wessman <https://github.com/voxpelli>
```

Comment:
```suggestion
//                 Pelle Wessman <https://github.com/voxpelli>
//                 Jordan Harband <https://github.com/ljharb>
```

this way i'll get pinged on PRs automatically :-)